repair/repair.hh

```diff
@@ -251,6 +251,7 @@ public:
     void abort_all_repairs();
     semaphore& range_parallelism_semaphore();
     static size_t max_repair_memory_per_range() { return _max_repair_memory_per_range; }
+    seastar::gate& gate() { return _gate; };
```

Comment:
Following my comment above: this seems to be unsafe.

Reply:
It is safe, because _gate is accessed from shard zero only.

Reply:
I see. You could consider separating the "sharded" part of the tracker and the "shard0 only" part into separate data structures. This would make maintaining a lot easier. But this is for future patches.